Fixed errors in doc (#139)
yugabyte/terraform-provider-ybm: docs/resources/associate_metrics_exporter_cluster.md

### Optional

- `cluster_id` (String) ID of the cluster with which this metrics export configuration will be associated.
- `cluster_name` (String) Name of the cluster with which this metrics export configuration will be associated.

> - `config_id` (String) ID of the metrics export configuration. Created automatically when enabling metrics export.

Reviewer: ```suggestion
- `config_id` (String) ID of the integration configuration to enable/modify metrics export on a cluster.
```

> - `config_name` (String) The name of the metrics export configuration

Reviewer: ```suggestion
- `config_name` (String) The name of the integration configuration to enable/modify metrics export on a cluster.
```
